### PR TITLE
Update product-profiles.md

### PR DESCRIPTION
**Pull request by samgnaniah**
Fixes https://github.com/wso2-enterprise/wso2-apim-internal/issues/635

### CODE REVIEW

File: en/docs/install-and-setup/setup/distributed-deployment/product-profiles.md [hunk 1]
```diff
@@ -158,6 +158,9 @@ Create an optimized distribution for a particular API-M profile.
 
 #### Method 2 - Optimizing while starting the server
 
+!!! note
+    Before running these profiles, take the backup of `<PRODUCT-HOME>/repository/resources/conf/templates/repository/conf/axis2` and `<PRODUCT-HOME>/ repository/resources/conf/templates/repository/conf/registry_TM.xml.j2` directories.
```
Comment (PasanT9) [inline en/docs/install-and-setup/setup/distributed-deployment/product-profiles.md:162]: Shall we change this as

Before running these profiles, take the backup of `<PRODUCT-HOME>/repository/resources/conf/templates/repository/conf/axis2` directory and `<PRODUCT-HOME>/ repository/resources/conf/templates/repository/conf/registry_TM.xml.j2` file

File: en/docs/install-and-setup/setup/distributed-deployment/product-profiles.md [hunk 2]
```diff
@@ -220,6 +223,8 @@ Create an optimized distribution for a particular API-M profile.
     
     ```  
         
+If you have encountered any issue relevant to loading the axis2 configuration, check if all the files are available in the `<PRODUCT-HOME>/repository/resources/conf/templates/repository/conf/axis2` directory and it tallies with the backup directory. If not, please add the missing J2 files accordingly.
```
Comment (PasanT9) [inline en/docs/install-and-setup/setup/distributed-deployment/product-profiles.md:226]: Shall we mention the <PRODUCT-HOME>/ repository/resources/conf/templates/repository/conf/registry_TM.xml.j2 file also ?

This is outside the axis2 directory.

Comment (samgnaniah) [reply en/docs/install-and-setup/setup/distributed-deployment/product-profiles.md:231]: Fixed with https://github.com/wso2/docs-apim/pull/6215/commits/0c0f0becf96542fbc506712404aef53efa128892

File: en/docs/install-and-setup/setup/distributed-deployment/product-profiles.md (hunk 1 above)
Comment (samgnaniah) [reply en/docs/install-and-setup/setup/distributed-deployment/product-profiles.md]: Fixed with https://github.com/wso2/docs-apim/pull/6215/commits/0c0f0becf96542fbc506712404aef53efa128892

**Review by PasanT9 — APPROVED** [review]

**Pull request merged**